### DISCUSSION TITLE
Gitea deployment error

### DISCUSSION THREAD

**Discussion by Cookie1990** [tteck/Proxmox#3488] [General]
Hello,
I just tried deploying the Gitea Server.
It seems thou that only Gitea is installed, not the necessary Database?
See Picture provided:

![grafik](https://github.com/user-attachments/assets/58050dcc-1ebb-4fdf-aa83-07a12a281e01)

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/blob/6d1cc92b59482b4d8dca0f9840f13eed41dbe57e/install/gitea-install.sh#L22
